src/win32/mouse.c

```diff
@@ -20,13 +20,30 @@
 	: ((button) == RIGHT_BUTTON ? MOUSEEVENTF_RIGHTDOWN \
 	: MOUSEEVENTF_MIDDLEDOWN))
 
+int CalculateAbsoluteCoordinateX(int x) {
+	MMSize displaySize = getMainDisplaySize();
+	return (x * 65536) / displaySize.width;
+}
+
+int CalculateAbsoluteCoordinateY(int y) {
+	MMSize displaySize = getMainDisplaySize();
+	return (y * 65536) / displaySize.height;
+}
+
```

Comment:
**Suggestion:** `moveMouse` receives an `MMPoint` struct, so we could merge these to methods to receive and return an `MMPoint` as well. This way we would also only have to call `getMainDisplaySize` once.